flow-plugins/vaadin-gradle-plugin/build.gradle

```
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2018-2019 Devsoap Inc.
+ * Copyright 2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/***********************************************************************************************************************
+ *
+ * Plugins
+ *
+ **********************************************************************************************************************/
+plugins {
+    id 'java-gradle-plugin'
+	id 'maven-publish'
+    id 'idea'
+    id 'com.gradle.plugin-publish' version '0.11.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.31'
+    id 'maven'
+}
+
+/***********************************************************************************************************************
+ *
+ * Basic properties
+ *
+ **********************************************************************************************************************/
+
+def pom = new XmlSlurper().parse(rootProject.file('pom.xml'))
+defaultTasks 'jar'
+group '${pom.parent.groupId}'
+version = project.hasProperty('version') ? getProperty('version') : '20.0-SNAPSHOT'
```

```suggestion
group pom.parent.groupId
version = project.hasProperty('vaadin.version') ? project.getProperty('vaadin.version') : pom.parent.version
```

`version` seems to always be undefined so it always exists as a property.